components/webui/imports/api/search/server/publications.js

```diff
@@ -66,7 +68,9 @@ Meteor.publish(Meteor.settings.public.SearchResultsCollectionName, ({
         ],
         limit: SEARCH_MAX_NUM_RESULTS,
         disableOplog: true,
```

Comment:
Why don't we toggle this according to `isActivelyPolling`?

Reply:
`disableOplog` currently is not doing anything. If we want to use the polling workaround, it is better to make `disableOplog` to be true.

Reply:
Let's document (add a comment in code to explain) why `disableOpLog` is unavailable: https://github.com/meteor/meteor/blob/7e2abe9464c215aace47d20f737e59243a819871/packages/mongo/oplog_observe_driver.js#L977

It might be good to rename `isActivelyPolling` -> `isExpectingUpdate`. @kirkrodrigues What's your take on this?

Reply:
isExpectingUpdate**s** sgtm

Reply:
Sorry, I have misread the Meteor code. Their docs do say "limit **w/o** a sort specifier is not supported":
1. L71: here we do specify BOTH sort and limit.
2. L97: here we specify neither sort or limit.

The cause of that setting only `disableOplogs: true` still yields high CPU usage is unknown. I can take a look when I pull and verify the changes. For now, let's remove the comments at L71 & L98 since neither matches what we specify in the find options.